common/coat_of_arms/coat_of_arms/wc_dynasties.txt

```diff
@@ -58,6 +58,15 @@ house_dragonmaw = 2401
 2250 = d_burning_blade
 house_burningblade = d_burning_blade
```

Comment:
Don't you want to make the Hellscream dynasty use the Warsong symbol?

Reply:
I prefer to use this kind of flags for dynasties that have the same name as the clan.
As for "of Frostwolf" which has the same flag as the clan. The same goes for the "of Blackrock" dynasty which was replaced by an original flag for Blackhand.